dbt/seeds/etl_full_row_counts.csv

```diff
@@ -304,7 +304,7 @@ core_eia860__scd_plants,2021,13490
 core_eia860__scd_plants,2022,14274
 core_eia860__scd_plants,2023,15108
 core_eia860__scd_plants,2024,15993
-core_eia860__scd_plants,2025,16576
+core_eia860__scd_plants,2025,16688
```

Comment:
the gha run # for this is 16689

Reply:
Hmm that seems weird.  How are you grabbing the row counts from the branch build outputs?

I checked for differences by copying down the row counts from the branch build into my `dbt/seeds` directory:

```console
cd dbt/seeds
gcloud storage cp gs://builds.catalyst.coop/2025-08-12-1614-314a6e6e4-update-eia860m-may-june-2025/etl_full_row_counts.csv .
```

and looked at `git diff` output. The only changes I saw were to:
* EPA CEMS in 2018, 2023, 2024 and 2025
* A bunch of FERC-714 tables gained 2024 row counts that they didn't have before.